### PR TITLE
[15.0][ADD] website_sale_stock

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: website_sale_stock/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "author": "Nitrokey GmbH, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/website",
+    "summary": """
+This module adds a new option to products to control how (almost) empty stock
+is displayed in the online shop. The new option is: "show inventory below a"
+threshold, sell regardless and show product-specific notification for empty
+"stock"
+""",
+    "depends": [
+        "website_sale_stock",
```
Comment (tarteo) [inline website_sale_stock/__manifest__.py:15]: It depends on itself

File: website_sale_stock/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Nitrokey Website Sale Stock - Website Delivery Information",
```
Comment (tarteo) [inline website_sale_stock/__manifest__.py:2]: Please, remove branding from the module name

File: website_sale_stock/static/src/js/website_sale_stock.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        qty = combination.free_qty || 1;
+        $addQtyInput.val(qty);
+      }
+      if (combination.free_qty < 1) {
+        $parent.find("#add_to_cart").addClass("disabled out_of_stock");
+        $parent.find(".o_we_buy_now").addClass("disabled out_of_stock");
+      }
+    }
+    var parent_xml = ajax.loadXML(
+      "/website_sale_stock/static/src/xml/website_sale_stock_product_availability.xml",
```
Comment (tarteo) [inline website_sale_stock/static/src/js/website_sale_stock.js:52]: This makes it load the xml everytime the product combination changes. Please do this only once

**Review by tarteo — APPROVED** [review]
This is module already exists in Odoo CE

image

**Review by tarteo — CHANGES_REQUESTED** [review]
I meant to request changes

**Comment by pedrobaeza** [comment]
Has this module disappears in Odoo core with no replacement? If copied from Odoo, the author should be Odoo SA and OCA, a disclaimer should be added on the readme, and then license should be respected.

File: website_sale_stock/static/src/js/website_sale_stock.js (hunk 3 above)
Comment (dsolanki-initos) [reply website_sale_stock/static/src/js/website_sale_stock.js:52]: > This makes it load the xml everytime the product combination changes. Please do this only once

Sorry and thank you for reviewing it. I am closing this PR and will create new one as per your suggested changes.

**Comment by dsolanki-initos** [comment]
> Has this module disappears in Odoo core with no replacement? If copied from Odoo, the author should be Odoo SA and OCA, a disclaimer should be added on the readme, and then license should be respected.

Thanks for the review there is such exist in odoo core and I am closing this PR as it conflict the name with odoo core. I will create new PR.

**Pull request closed without merging**